os/binfmt/binfmt_loadbinary.c

```diff
@@ -211,14 +211,15 @@ int load_binary(int binary_idx, FAR const char *filename, load_attr_t *load_attr
 	heap_table[binary_idx] = bin->heapstart;
 #endif
 
+	elf_save_bin_section_addr(bin);
```

Comment:
Is there any chance to use the app text address even though that app is not executed well?
If execution fails, that app doesn't do anything.

Reply:
Hello @jeongchanKim , these changes are for the case where the app crashes before saving the text address (as the app execution has started before calling elf_save_bin_section_addr) and thereby not getting the app text address in the crash logs. This is for the case when exec_module has passed but the app has crashed before saving the text addresses.

Reply:
Oh executed app can have higher priority.
Agreed. Thanks.

Reply:
> Oh executed app can have higher priority. Agreed. Thanks.

yes. 
Can you please test easysetup crash issue with this patch on product_refrigerator